src/App.js

```diff
@@ -1,26 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './Components/Calculator';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <Calculator />
       </header>
     </div>
   );
```

Comment:
- [ ] it would be better if you change this component from function to class-based as in the requirements
```JAVASCRIPT

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```